rhel/SPECS/victoriametrics.spec

```diff
@@ -7,17 +7,21 @@
 %doc %{*} \
 %endif
 
+%global provider        github
```

Comment:
I though that we are going to simplify all of these stuff

Reply:
Fixed